ui/app/manager/src/pages/page-configuration.ts

```diff
@@ -238,17 +316,22 @@ export class PageConfiguration extends Page<AppStateKeyed> {
                         ></or-conf-json>
                     </div>
                 `;
+                const mapHeading = html`
+                    <div id="heading" style="justify-content: space-between;">
+                        <span style="margin: 0;"><or-translate style="text-transform: uppercase;" value="configuration.mapSettings"></or-translate></span>
+                    </div>
+                `;
                 const realmOptions = this.realms?.map((r) => ({name: r.name, displayName: r.displayName, canDelete: true}));
                 realmOptions.push({name: 'default', displayName: 'Default', canDelete: false});
                 return html`
                     <div id="wrapper">
                         <div id="header-wrapper">
                             <div id="header-title">
                                 <or-icon icon="palette-outline"></or-icon>
-                                ${i18next.t("appearance")}
+                                <or-translate value="appearance"></or-translate>
                             </div>
                             <div id="header-actions">
-                                <or-mwc-input id="save-btn" .disabled="${!this.managerConfigurationChanged && !this.mapConfigChanged}" raised type="button" label="save"
+                                <or-mwc-input id="save-btn" .disabled="${!this.managerConfigurationChanged && !this.mapConfigChanged && !this.tilesForUpload && !this.tilesForDeletion}" raised type="button" label="save"
```

Comment:
For readability purposes, move this long statement from inline-JS to a small `const disabled = ... ;` line.